### PR TITLE
Fixes for documentation

### PR DESCRIPTION
**Pull request by dabacon**
First page of searching for #3388 TODOs.

Non-trivial changes:
* AQT was making extra isinstance checks on operations to be of a gate type.

### CODE REVIEW

File: cirq-aqt/cirq_aqt/aqt_device.py [hunk 1]
```diff
@@ -24,43 +24,43 @@
...
-# pylint: disable=missing-raises-doc
 def get_op_string(op_obj: cirq.Operation) -> str:
-    """Find the string representation for a given gate
+    """Find the string representation for a given gate or operation.
 
     Args:
-        op_obj: Gate object, one of: XXPowGate, XPowGate, YPowGate, ZPowGate,
-            PhasedXPowGate
+        op_obj: Gate or operation object. Gate must be one of: XXPowGate, XPowGate, YPowGate,
+        ZPowGate, PhasedXPowGate, or MeasurementGate.
```
Comment (MichaelBroughton) [inline cirq-aqt/cirq_aqt/aqt_device.py:39]: Nit: more indentation here.

File: cirq-google/cirq_google/api/v2/results.py [hunk 2]
```diff
@@ -198,6 +197,9 @@ def _trial_sweep_from_proto(
 
     Returns:
         A list containing a list of trial results for the sweep.
+
+    Raises:
+        ValueError: If a qubit repeats in the measurement results.
```
Comment (MichaelBroughton) [inline cirq-google/cirq_google/api/v2/results.py:202]: Clarification nit: "Already exists."

File: cirq-google/cirq_google/engine/engine.py [hunk 3]
```diff
@@ -800,8 +811,8 @@ def get_engine(project_id: Optional[str] = None) -> Engine:
         The Engine instance.
 
     Raises:
-        EnvironmentError: If the environment variable GOOGLE_CLOUD_PROJECT is
-            not set.
+        OSError: If the environment variable GOOGLE_CLOUD_PROJECT is not set. This is actually
+            an `EnvironmentError`, which by definition is an `OsError`.
```
Comment (MichaelBroughton) [inline cirq-google/cirq_google/engine/engine.py:815]: Technically correct, the best kind of correct.

**Review by MichaelBroughton — APPROVED** [review]
LGTM

File: cirq-google/cirq_google/engine/engine.py (hunk 3 above)
Comment (dabacon) [reply cirq-google/cirq_google/engine/engine.py:815]: Lint yells if you call it an `EnvironmentError`, so this is the annoying workaround.

**Pull request merged**